qiskit/schemas/qobj_schema.json

```diff
@@ -273,10 +273,12 @@
                                 "parametric_pulse"
                             ]
                         },
+                        "label": {"type": "string"},
                         "pulse_shape": {"type": "string"},
                         "parameters": {"type": "object"}
                     },
                     "required": [
+                        "label",
```

Comment:
This should not be required as it will break on devices that are already deployed.